docs/examples.rst

```diff
@@ -306,3 +306,56 @@ Output:
+
+    # Address field should be the checksum address at which the ERC20 contract was deployed
+    erc20 = w3.eth.contract(address='0x89205A3A3b2A69De6Dbf7f01ED13B2108B2c43e7', abi=ERC20_ABI)
+
+Fetch various data about the current state of the ERC20 contract.
+
+.. code-block:: python
+
+    name = erc20.functions.name().call()
+    # -> "Unicorns"
+    symbol = erc20.functions.symbol().call()
+    # -> "🦄"
+    decimals = erc20.functions.decimals().call()
+    # -> 1
+    total_supply = erc20.functions.totalSupply().call()
+    # -> 2038 (at time of writing)
+
+Get the balance of an account address. 
+
+.. code-block:: python
+
+    balance = erc20.functions.balanceOf('0xD1220A0cf47c7B9Be7A2E6BA89F429762e7b9aDb').call()
+    # -> 1
+
+Calculate the token count from the token balance of an account address.
+
+.. code-block:: python
+   
+    from decimal import Decimal
+    
+    # Most applications expect *exact* results, so using the Decimal library,
+    # with default precision of 28, is usually sufficient to avoid any rounding error.
+    count = balance / Decimal(10 ** decimals)
+    # -> 1
+
+Balance is *always* an integer in the currency's smallest natural unit (i.e. `wei` for ether). Token balance is typically used only for backend calculations.
+
+Token count *may* be a integer or fraction in the currency's primary unit (i.e. `eth` for ether). Token count is typically displayed to the user on the front-end since it is more readable.
```

Comment:
Instead of `i.e.` (which means "in other words"). Since ether isn't an ERC20 token, this could be confusing. I would say something like `(equivalent to 'eth' for ether)`